NitroxServer/Communication/LANDiscoveryServer.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Net;
+using System.Threading;
+using LiteNetLib;
+using LiteNetLib.Utils;
+using NitroxModel.Constants;
+
+namespace NitroxServer.Communication
+{
+    public static class LANDiscoveryServer
+    {
+        private static NetManager server;
+        private static EventBasedNetListener listener;
+
+        private static Timer pollTimer;
+
+        public static void Start()
+        {
+            listener = new EventBasedNetListener();
+            server = new NetManager(listener);
+
+            server.AutoRecycle = true;
+            server.DiscoveryEnabled = true;
+            server.UnconnectedMessagesEnabled = true;
+
+            for (int i = 0; i < LANDiscoveryConstants.BROADCAST_PORTS.Length; i++)
+            {
+                if (server.Start(LANDiscoveryConstants.BROADCAST_PORTS[i]))
+                {
+                    break;
+                }
+            }
```

Comment:
Should we break at the first one started? I can imagine a "very professional" user network where client and server start their NetManager at different ports. Would it be possible to start the 5 servers and would it impact performance?

Reply:
Good point. Luckily I don't think starting multiple NetManagers is necessary because the client can send a discovery request to all of the ports instead of just the one it is listening on.